projects/fsastorefront-e2e-cypress/cypress/integration/checkout/autoe2e/autoGold_Card_AddDriver_RemoveCoverage.e2e.ts

```diff
@@ -0,0 +1,113 @@
+import { registrationUser } from '../../../sample-data/users';
+import * as register from '../../../helpers/register';
+import * as auto from '../../../helpers/checkout/insurance/auto';
+import * as autoIntegration from '../../../helpers/checkout/insurance/autoIntegration';
+import * as checkout from '../../../helpers/checkout/checkoutSteps';
+import { selectPaymentMethodInvoice } from '../../../helpers/checkout/insurance/payment';
+import * as myPolicies from '../../../helpers/my-account/policies';
+import * as changeRequest from '../../../helpers/changeRequest';
+const todaysDate = Cypress.moment().format('YYYY-MM-DD');
+
+Cypress.config('defaultCommandTimeout', 500000);
+
+context('Auto Gold Checkout with Adding Driver and Removing Coverage', () => {
+  before(() => {
+    cy.visit('/');
+  });
+
+  it('Should register a new user', () => {
+    register.registerUser(registrationUser);
+    register.login(registrationUser.email, registrationUser.password);
+  });
+
+  it('Should complete first auto step without additional drivers', () => {
+    checkout.waitConsent();
+    checkout.startInsuranceCheckout('Auto');
+    auto.populateAutoMonthlyAudi();
+    auto.populateMainDriverInfo();
+    cy.get('[name=noOfDrivers]').select('1');
+    auto.populateAdditionalDriverInfo();
+    checkout.clickContinueButton();
+  });
+
+  it('Should check comparison table and select Gold Product', () => {
+    autoIntegration.selectAutoGold();
+    auto.checkOptionalProductsGoldAddOptional();
+    checkout.clickContinueButton();
+  });
+
+  it('Should populate personal details page', () => {
+    checkout.checkPersonalDetailsPage();
+    auto.populatePersonalDetails();
+    auto.populateVehicleDetails();
+    auto.populateMainDriverData();
+    auto.populateAdditionalData();
+    cy.get('.primary-button').should('be.visible');
+    checkout.clickContinueButton();
+  });
+
+  it('Should bound a quote', () => {
+    checkout.checkCheckoutStep('Your Auto Insurance', '7');
+    checkout.checkProgressBarInsurance();
+    checkout.checkAccordions('generalQuoteAccordions');
+    cy.get('.primary-button').should('contain.text', 'Continue');
+    checkout.clickContinueButton();
+    checkout.ConfirmBindQuote();
+    checkout.clickContinueButton();
+  });
+
+  it('Select default payment details and place an order', () => {
+    //TODO: Will be replaced with credit card once bug is resolved
+    selectPaymentMethodInvoice();
+    checkout.clickContinueButton();
+    checkout.placeOrderOnFinalReview();
+    checkout.checkOrderConfirmation();
+  });
+
+  it('Should check my policies and policy details page', () => {
+    //waiting for replication process to be completed
+    cy.wait(200000);
```

Comment:
i guess, it's better to mock some policy than to actually create policy in external system and wait for replication :)

Reply:
I mean, this kind of tests should be used to actually test certain functionality but again, in this particular test we are waiting  400 seconds and we actually need it bcs of policy replication.  I don't know what to do here, do we really need to test the integration use-case? The only difference is waiting for policy replication in our system

Reply:
Waiting for policy replication is only for integration scenarios (auto checkouts) and needed since integrations are not stable enough. If we remove replication we cannot be sure it is working end to end